packages/debug/src/node/vscode/vscode-debug-adapter-contribution.ts

```diff
@@ -170,16 +170,18 @@ export abstract class AbstractVSCodeDebugAdapterContribution implements DebugAda
             };
             properties['preLaunchTask'] = {
                 anyOf: [taskSchema, {
-                    type: ['string', 'null'],
+                    type: ['string'],
                 }],
                 default: '',
+                defaultSnippets: [{ body: { task: '', type: '' } }],
```

Comment:
could you please give me an example of how `{ task: '', type: '' }` can be used?

Reply:
copy/paste from VS Code. 
https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/contrib/debug/common/debugger.ts#L242